common/djangoapps/student/tests/test_create_account.py

```diff
@@ -1,8 +1,10 @@
 """Tests for account creation"""
 import json
-
+from datetime import datetime
 import ddt
+from mock import patch
 import unittest
+
```

Comment:
Minor: since you're making changes here, please correct the import order (one block each for standard library, third-party, and first-party imports). Also, Mock is already imported below; it, along with ddt, should be part of a second block of third-party imports.